community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/path/DijkstraBidirectional.java

```diff
@@ -174,7 +175,7 @@ public TraversalMetadata metadata()
             if ( NoneStrictMath.compare( thisState + otherSideShortest.doubleValue(), shortestSoFar.doubleValue(), epsilon ) > 0 &&
                  stopAfterLowestCost )
             {
-                return Collections.emptyList();
+                return Iterables.resourceIterable( Collections.emptyList() );
```

Comment:
There's an `Iterators#emptyResourceIterator` method, perhaps we should introduce a `Iterables#emptyResourceIterable` too?